### DISCUSSION TITLE
Highlight glossary terms in secondary source strings

### DISCUSSION THREAD

**Discussion by parsifino** [WeblateOrg/weblate#8665] [Q&A]
Hi all,

we have ITALIAN as the source language and we translate to several languages (English, German, French, etc..)

We have some translators that understand english but not Italian, so they use English as a secondary language to translate from.

However, they don't have access to the glossary, because it looks like it's ignored, even if I create another glossary with English as the source.

See picture.
I believe the secondary language (english in my case) should have highlights. Is there a way to fix this?

![image](https://user-images.githubusercontent.com/20045013/214258366-75cce247-e406-4979-abf5-d11e11e5cdca.png)

**Comment by nijel** [comment]
The highlighting is based on the source term in glossary, which is in Italian, why it should be matched against English string?

**Reply by parsifino** [reply]
Thank you.

You're right: I should clarify this. I created a secondary glossary which uses English as source. However it is ignored: it looks like Weblate only checks for glossaries that have the same source language as the component that I am translating (Italian in my case), even if the translator uses a secondary language.

I believe it would be useful to highlight the secondary language, especially when the translator flags the option to hide the source language when a translation in the second language exists.

**Reply by nijel** [reply]
Only glossary with matching source language is used when browsing the component.
